components/quasar_s3_export_role/main.tf

```diff
@@ -0,0 +1,67 @@
+resource "aws_s3_bucket" "dosomething_quasar_archive" {
+  bucket = "dosomething-quasar-archive"
```

Comment:
On some further reflection, I think it does make sense to put these in a separate bucket since these files will be parquet files and may end up being the beginning of our data lake. If we want to query the snapshots we'll probably use something like Athena, so it would make sense to separate the snapshot files from other types of backup files.

Reply:
Ah, that makes great sense to me!